client/src/assets/css/main.css

```diff
@@ -0,0 +1,48 @@
+.text-center{
+    text-align: center;
+}
+
+.text-right{
+    text-align: right;
```

Comment:
Yea this is really standard. Generally discouraged from using inline css and general classes make for better reuse.